src/main/java/frc/robot/Robot.java

```diff
@@ -72,7 +71,7 @@ private void configureBindings() {
     controller.circle().whileTrue(swerve.pathFindToLocation(HM_RIGHT));
 
     //Intake:
-    intake.setDefaultCommand(intake.operateIntakeCommand(operator::getR2Axis, operator::getL2Axis));
+    intake.setDefaultCommand(intake.manualIntakeCommand(operator::getR2Axis, operator::getL2Axis));
```

Comment:
at the moment there are two defualt commands for the intake, delete this one (not the manual command - the fact that you set it as the defualt command